docker-compose.env.sample

```diff
@@ -4,3 +4,4 @@ DATABASE_URL=postgres://postgres@db:5432/tvs_development?template=template0&pool
 REDIS_URL=redis://redis:6379
 AAD_CLIENT_ID=
 AAD_TENANT=
+DOMAIN=localhost:3000
```

Comment:
can we use `domain = URI(ENV['DOMAIN'])` in development and test since we do set the `DOMAIN` variable?

Reply:
Not sure I understand the purpose of changing to `domain = URI(ENV['DOMAIN'])` however you are right in that the value it not needed in dev or test given the config in staging.rb and production.rb, so perhaps it's not useful and in fact should be removed.